PiwikTracker/Context.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+
+public class Context {
```

Comment:
So we make two structs 'Device' and 'Application' not nested in a 'Context'?! What do you think @brototyp ?